documentation/staging/content/samples/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -301,7 +294,7 @@ We need to set up the domain configuration for the WebLogic domain.
    troubleshoot the reason and resolve it before proceeding to the next
    step.
 
-    {{% notice note %}} This sample creates WebLogic Server Pods with reasonable values for memory, CPU, and JVM heap size (as a percentage of memory). These settings were determined by running a skeleton WebLogic domain with minimal or no deployed services and applications on potentially limited or heavily shared container environments. For advice about tuning CPU and memory requests and limits for broader use cases or in a production environment, see the [Pod memory and CPU resources](https://oracle.github.io/weblogic-kubernetes-operator/faq/resource-settings/) FAQ. To supply different values, edit `~/azure/weblogic-on-aks/domain1.yaml` and set the desired values for `serverPodMemoryRequest`, `serverPodMemoryLimit`, `serverPodCpuRequest`, `serverPodCpuLimit` and `javaOptions` before running `./create-domain.sh -i ~/azure/weblogic-on-aks/domain1.yaml -o ~/azure -e -v`.
+    {{% notice note %}} This sample creates WebLogic Server pods with reasonable values for memory, CPU, and JVM heap size (as a percentage of memory). You can supply different values. Edit `~/azure/weblogic-on-aks/domain1.yaml` and set the desired values for `serverPodMemoryRequest`, `serverPodMemoryLimit`, `serverPodCpuRequest`, `serverPodCpuLimit` and `javaOptions` before running `./create-domain.sh -i ~/azure/weblogic-on-aks/domain1.yaml -o ~/azure -e -v`.
```

Comment:
The new notice seems to revert capacity planning guidance text that we all agreed to earlier.  Was this done on purpose?

Reply:
No, I copied them from 3.3 doc, I found you have a commit https://github.com/oracle/weblogic-kubernetes-operator/commit/07660ee8d76f3de7896433aa2f60229c97d7ccdc, and would like to keep the two documents the same.

Do we apply the new text for 3.3 doc?

Reply:
ah, I copied the content from [3.3 doc ](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/3.3/content/samples/azure-kubernetes-service/domain-on-pv.md), will rec=vert them, and also applied the changes to /3.3/content/samples/azure-kubernetes-service/domain-on-pv.md.

Reply:
See https://github.com/galiacheng/weblogic-kubernetes-operator/commit/8cdad288498734cfb52b50af88d61387741fa386

Reply:
Yes, please apply my edits/comments to both docs. Thank you.

Reply:
Sure @rosemarymarano please have a look at this commit https://github.com/oracle/weblogic-kubernetes-operator/pull/2727/commits/dfddc4c7b731dfd5c2e1bcb81201205c91f5f81e?